tests/python/openassetio/managerApi/test_managerinterface.py

```diff
@@ -17,7 +17,7 @@
 Tests for the default implementations of ManagerInterface methods.
 """
 
-# pylint: disable=no-self-use
+# pylint: disable=no-self-use, too-few-public-methods
```

Comment:
[question] The extra space after the `,` made me curious (turns out pylint doesn't mind spaces, pretty sure it used to).  Anyway, I removed this change and ran pylint and got no complaints. Looking at `pyproject.toml` we explicitly disable `too-few-public-methods` globally, so this linter warning should never have been flagged.  Is this possibly a local config issue, or did it crop up on CI?

Reply:
Ahh interesting, it was noticed locally. Ill see why 🤔 thanks.